src/routes/changelog/(entries)/2025-01-03.markdoc

```diff
@@ -0,0 +1,46 @@
+---
+layout: changelog
+title: Billing improvements, UI fixes, and performance updates
+date: 2025-01-03
+---
+
+We've shipped several improvements to make Appwrite easier to use, including a clearer billing system, UI updates, and fixes for collections and mobile experiences.
+
+# What's new
+
+- Premium support checks now rely on a backend plan attribute instead of manual frontend validation
```

Comment:
not sure if this adds any value to readers, as this is more of an internal fix. but no strong opinion/preferences though.